src/Collection.php

```diff
@@ -93,4 +94,37 @@ static function ($item) {
             }
         );
     }
+
+    /**
+     * Add items that are not in the collection but are intermediate nodes
+     */
+    public function fillMissingIntermediateNodes(): void
+    {
+        $nodeIds    = $this->pluck('id', 'id')->all();
+        $collection = $this->sortByDesc(static fn($item) => $item->levelValue());
+
+        foreach ($collection as $node) {
+            if ($node instanceof Model && !$node->isRoot() && !(isset($nodeIds[$node->parentValue()]) ?? false)) {
+                $parents = $node->parents()
+                    ->filter(
+                        static fn(Model $model) => !(isset($nodeIds[$node->parentValue()]) ?? false)
+                    );
+
+                $this->items = array_merge($this->items, $parents->all());
+                $nodeIds     = array_merge($parents->pluck('id', 'id')->all(), $nodeIds);
+            }
+        }
+    }
+
+    /**
+     * @param Model|string|int|null $fromNode
+     *
+     * @return $this
+     */
+    public function toBreadcrumbs(Model|string|int|null $fromNode = null): static
+    {
+        $this->fillMissingIntermediateNodes();
+
+        return $this->toTree($fromNode);
+    }
```

Comment:
let's create a static method to create CollectionTree from any Nodes. For examples: 

`Collection::makeFromNodes([$model1, $model421]);`
`Collection::makeFromNodes($model1]);`